beacon-chain/blockchain/process_block.go

```diff
@@ -245,7 +302,57 @@ func (s *Service) onBlockInitialSyncStateTransition(ctx context.Context, signed
 		}
 	}
 
-	// Epoch boundary bookkeeping such as logging epoch summaries.
+	return s.handleEpochBoundary(postState)
+}
+
+// handles a block after the block's batch has been verified, where we can save blocks
+// their state summaries and split them off to relative hot/cold storage.
+func (s *Service) handleBlockInBatch(ctx context.Context, signed *ethpb.SignedBeaconBlock,
```

Comment:
```suggestion
func (s *Service) handleBlockAfterBatchVerify(ctx context.Context, signed *ethpb.SignedBeaconBlock,
```

`handleBlockInBatch` is not accurate since we only pass in a block as argument

Reply:
alright done